### DISCUSSION TITLE
Questions & Answers

### DISCUSSION THREAD

**Discussion by ronyeh** [0xfe/vexflow#1165] [Q&A]
Go to the Q&A tab and start a new discussion:
https://github.com/0xfe/vexflow/discussions/categories/q-a

Or, if you have a tiny question, feel free to drop it below and maybe one of us will see it ;-).

--- 

Here are some other questions I've run across:

### [ramezzi](https://github.com/ramezzi) asked on Jan 27
Hi everyone! I was wondering if there is a way to add notes to an existing stave and change the pitch of those notes after adding them? I couldn't find anything on this specifically so I proceeded to ask here. All help is greatly appreciated!

### More to come... 
I will collect more questions that I run across and add them here. If you have a good answer, please post below!

**Comment by AaronDavidNewman** [comment]
@ronyeh, @rvilarl I have a question for you guys.  How is it you are able to submit multiple PRs in parallel?

I read when I started this project that GH does not support this for public repositories.  I would have to create more than one GH account if I wanted to have multiple PRs flowing into the same project.  But you are clearly not doing that, and I see multiple outstanding.  I must be missing something obvious, the obvious is never so obvious for me it seems.

This may not be a good forum for it, but I didn't want to open an issue for a process question that doesn't directly relate to vexflow.

**Comment by ronyeh** [comment]
My master branch tracks 0xfe's master branch. I create a different branch for each feature. Then I submit PRs from my feature branch.

**Comment by mscuthbert** [comment]
Use different branches.  But be sure to restart grunt after switching if any of the Interfaces have changed.
